src/main/java/gov/usds/case_issues/CaseIssueApi.java

```diff
@@ -3,9 +3,11 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cache.annotation.EnableCaching;
 
 @SpringBootApplication
 @EnableConfigurationProperties
+@EnableCaching
```

Comment:
I suspect this is a short-term solution: when we have multiple server instances, simple in-memory cache seems dangerous, and we are unlikely to want to put a more robust cache implementation (e.g. redis) in place just for this.

Reply:
Added this to the backlog https://trello.com/c/Awm0vtpF also created a new Epic to include the tech debt that needs to be solved for us to scale the backend to more than one node.

Added the in memory user sessions to the Epic as well https://trello.com/c/Zp5iiahW